<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Semibold</family>
    <pointsize>12</pointsize>
    <bold>true</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1600</width>
     <height>900</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>63</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>63</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>63</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>63</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Camera" name="camer_output" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="confidence_slider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Accent">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>70</red>
                <green>180</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Accent">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>70</red>
                <green>180</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Accent">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>70</red>
                <green>180</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Semibold</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
            <underline>false</underline>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>95</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QLineEdit" name="confidence_percent">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Confidence:</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>264</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="end_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>132</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>49</green>
              <blue>49</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>70</red>
              <green>180</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>49</green>
              <blue>49</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>70</red>
              <green>180</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>49</green>
              <blue>49</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="run_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>132</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>49</green>
              <blue>49</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>70</red>
              <green>180</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>49</green>
              <blue>49</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>70</red>
              <green>180</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>49</red>
              <green>49</green>
              <blue>49</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>33</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuwhere_is_the_box">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionChange_Camera"/>
   </widget>
   <widget class="QMenu" name="menuDatasets">
    <property name="title">
     <string>Datasets</string>
    </property>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuCreateModels">
    <property name="title">
     <string>Create Model</string>
    </property>
    <addaction name="actionOpen_Annotator"/>
    <addaction name="actionTrain_Model_2"/>
   </widget>
   <widget class="Github" name="menuGithub">
    <property name="title">
     <string>Github</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLoad_Model">
    <property name="title">
     <string>Load Project</string>
    </property>
    <addaction name="actionSelect_Project_Directory"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_ReadME"/>
   </widget>
   <addaction name="menuwhere_is_the_box"/>
   <addaction name="menuDatasets"/>
   <addaction name="menuCreateModels"/>
   <addaction name="menuLoad_Model"/>
   <addaction name="menuGithub"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionWhy">
   <property name="text">
    <string>Why</string>
   </property>
  </action>
  <action name="actionDo">
   <property name="text">
    <string>Do</string>
   </property>
  </action>
  <action name="actionI">
   <property name="text">
    <string>I</string>
   </property>
  </action>
  <action name="actionHate">
   <property name="text">
    <string>Hate</string>
   </property>
  </action>
  <action name="actionMyself">
   <property name="text">
    <string>Myself</string>
   </property>
  </action>
  <action name="action_Yes">
   <property name="text">
    <string>%Yes</string>
   </property>
  </action>
  <action name="action_Yes_2">
   <property name="text">
    <string>&amp;Yes</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="actionChange_Camera">
   <property name="text">
    <string>Change_Camera</string>
   </property>
  </action>
  <action name="actionRemove_2">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAdd_2">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
  <action name="actionOpen_Annotator">
   <property name="text">
    <string>Open Annotator</string>
   </property>
  </action>
  <action name="actionSelect_Files">
   <property name="text">
    <string>Select Files</string>
   </property>
  </action>
  <action name="actionTrain_Model">
   <property name="text">
    <string>Train Model</string>
   </property>
  </action>
  <action name="actionTrain_Model_2">
   <property name="text">
    <string>Train Model</string>
   </property>
  </action>
  <action name="actionFrom_Project_Directory">
   <property name="text">
    <string>From Project Directory</string>
   </property>
  </action>
  <action name="actionSelect_Project_Directory">
   <property name="text">
    <string>Select Project Directory</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionOpen_ReadME">
   <property name="text">
    <string>Open ReadME</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Camera</class>
   <extends>QWidget</extends>
   <header>Camera.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Github</class>
   <extends>QMenu</extends>
   <header>Github.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>confidence_slider</tabstop>
  <tabstop>confidence_percent</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>run_button</sender>
   <signal>clicked()</signal>
   <receiver>camer_output</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>end_button</sender>
   <signal>clicked()</signal>
   <receiver>camer_output</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confidence_percent</sender>
   <signal>textEdited(QString)</signal>
   <receiver>confidence_slider</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
